raiden_installer/base.py

```diff
@@ -120,7 +117,7 @@ def get_available_configurations(cls):
     def load(cls, file_path: Path):
         file_name, _ = os.path.splitext(os.path.basename(file_path))
 
-        _, _, network_name = file_name.split("-")
+        _, _, settings_name = file_name.split("-")
```

Comment:
I agree that passing the settings_name around is not optimal, but I don't see an easy solution that avoids it.

When we load an old configuration we need to know with which settings it has been created, because we can't use it if it is not matching the settings that the wizard is currently using. I think we shouldn't add it to the content of the file because the Raiden Client does not need this information. We could also compare the content of the file to the available settings, but there could be multiple settings which match the content of the file. Do you see any other mapping the configuration to a settings file?